packages/libs/react-ui/src/components/index.ts

```diff
@@ -24,7 +24,7 @@ export type { ITooltipProps } from './Tooltip';
 export type { ITrackerCardProps, ILabelValue } from './TrackerCard/TrackerCard';
 export type { ITreeProps } from './Tree';
 export type {
-  INavFooterRootProps,
+  INavFooterContainerProps,
```

Comment:
I think Root makes more sense since the subcomponent is named Root. What do you think?

Reply:
Yeah that works, I'll have NavHeader match it then in this PR.